Source/FieldSolver/SpectralSolver/SpectralAlgorithms/AvgGalileanAlgorithm.H

```diff
@@ -0,0 +1,30 @@
+#ifndef WARPX_AVG_GALILEAN_ALGORITHM_H_
+#define WARPX_AVG_GALILEAN_ALGORITHM_H_
+
+#include "FieldSolver/SpectralSolver/SpectralAlgorithms/SpectralBaseAlgorithm.H"
+
+/* \brief Class that updates the field in spectral space
+ * and stores the coefficients of the corresponding update equation.
+ */
+class AvgGalileanAlgorithm : public SpectralBaseAlgorithm
+{
+    public:
+        AvgGalileanAlgorithm(const SpectralKSpace& spectral_kspace,
+                         const amrex::DistributionMapping& dm,
+                         const int norder_x, const int norder_y,
+                         const int norder_z, const bool nodal,
+                         const amrex::Array<amrex::Real,3>& v_galilean,
+                         const amrex::Real dt);
```

Comment:
Here I think it would be better to add a space between the function name and the parentheses, following WarpX's style guidelines (this makes it easier to find function definitions with `git grep` separately from function calls):

```suggestion
        AvgGalileanAlgorithm (const SpectralKSpace& spectral_kspace,
                              const amrex::DistributionMapping& dm,
                              const int norder_x, const int norder_y,
                              const int norder_z, const bool nodal,
                              const amrex::Array<amrex::Real,3>& v_galilean,
                              const amrex::Real dt);
```

Reply:
I agree with @EZoni.

Reply:
Thanks for noticing! Fixed in commit 20f20b32732165e6f0b28c.